tests/test_views.py

```diff
@@ -1763,7 +1763,7 @@ def test_password_change_invalid_new_password_length(self):
             old_password='password',
             new_password='12345',
             confirm_password='12345',
```

Comment:
There is a test for valid password change that coincidentally has 8 character password. I added one specifically for 8 characters.